Makefile

```diff
@@ -511,6 +546,24 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+.PHONY: gcloud
+GCLOUD = ./bin/gcloud
```

Comment:
suggest using the gcloud docker image in place of the binary fetch.

Reply:
sure! let me give that a shot

Reply:
@benashz - I've changed this to a simple error message since this works primarily on GitHub actions where gcloud is already installed we'll leave it to other folks using this to install individually because the container + script approach doesn't work something as complex as gcloud where it sets up the docker credentials in the running machine